pkg/limayaml/defaults.go

```diff
@@ -24,9 +24,9 @@ func FillDefault(y *LimaYAML) {
 func resolveArch(s string) Arch {
 	if s == "" || s == "default" {
 		if runtime.GOOS == "amd64" {
```

Comment:
Lol, I made two typos (so it seemed to be working on Intel Mac):
- GOARCH as GOOS
- AARCH64 as X8664

Could you fix the GOARCH typo as well?